### PR TITLE
Fixed #25748 -- Added a table comparing projects and apps.

### PR DESCRIPTION
**Pull request by guettli**
https://code.djangoproject.com/ticket/25748

### CODE REVIEW

File: docs/ref/applications.txt [hunk 1]
```diff
@@ -41,6 +41,22 @@ interact with installed applications, mainly for configuration and also for
...
 :class:`~django.apps.AppConfig` instance for each installed application.
 
+Here is a comparison of "Project vs Application" for better understanding:
+
++--------------------+--------------------+------------------------------+
+| Topic              | Project            |Application                   |
++====================+====================+==============================+
+|Contains            | Settings           | Code, Views, Templates, ...  |
++--------------------+--------------------+------------------------------+
+|Size                | Only view lines    | Can be huge                  |
```
Comment (timgraham) [inline docs/ref/applications.txt]: Did you mean "view" -> "a few"?

File: docs/ref/applications.txt [hunk 2]
```diff
@@ -41,6 +41,22 @@ interact with installed applications, mainly for configuration and also for
...
++--------------------+--------------------+------------------------------+
+|Can have Models     | No                 | Yes                          |
```
Comment (timgraham) [inline docs/ref/applications.txt]: I don't think there's a restriction preventing a project from also being an application that has models.

File: docs/ref/applications.txt [hunk 3]
```diff
@@ -41,6 +41,22 @@ interact with installed applications, mainly for configuration and also for
 introspection. That's why the application registry maintains metadata in an
 :class:`~django.apps.AppConfig` instance for each installed application.
 
+Here is a comparison of "Project vs Application" for better understanding:
+
++--------------------+--------------------+------------------------------+
+| Topic              | Project            |Application                   |
++====================+====================+==============================+
+|Contains            | Settings           | Code, Views, Templates, ...  |
```
Comment (timgraham) [inline docs/ref/applications.txt]: The default project template also creates urls.py and wsgi.py. Some projects also store other things like templates and fixtures, see https://github.com/django/djangoproject.com/tree/master/djangoproject for an example.

File: docs/ref/applications.txt [hunk 4]
```diff
@@ -41,6 +41,22 @@ interact with installed applications, mainly for configuration and also for
...
++--------------------+--------------------+------------------------------+
+|Size                | Only view lines    | Can be huge                  |
++--------------------+--------------------+------------------------------+
+|Can have Models     | No                 | Yes                          |
++--------------------+--------------------+------------------------------+
+|May be re-usable    | No                 | Yes                          |
```
Comment (timgraham) [inline docs/ref/applications.txt]: no dash in "reuseable"

File: docs/ref/applications.txt [hunk 5]
```diff
@@ -41,6 +41,22 @@ interact with installed applications, mainly for configuration and also for
...
++--------------------+--------------------+------------------------------+
+
```
Comment (timgraham) [inline docs/ref/applications.txt]: only one blank line between sections please

File: docs/ref/applications.txt [hunk 6]
```diff
@@ -41,6 +41,22 @@ interact with installed applications, mainly for configuration and also for
 introspection. That's why the application registry maintains metadata in an
 :class:`~django.apps.AppConfig` instance for each installed application.
 
+Here is a comparison of "Project vs Application" for better understanding:
+
++--------------------+--------------------+------------------------------+
+| Topic              | Project            |Application                   |
```
Comment (timgraham) [inline docs/ref/applications.txt]: please be consistent with a space after each vertical bar

**Comment by timgraham** [comment]
I don't know that I like the table format for this information as they are a bit annoying to edit, but it could work.

**Comment by guettli** [comment]
Hi Tim Graham.

I updated the pull request and improved the formatting.

The cell Project/Contains was not changed, since I think "Settings" include root-urls, wsgi.py, ... Maybe this should be documented outside the table.

Please tell me if the pull request needs improvements.

**Comment by timgraham** [comment]
Here's my try at this: https://github.com/django/django/pull/5870.

**Comment by timgraham** [comment]
I merged my patch since I got some positive feedback from others. Feel free to follow up at your convenience if there are other improvements that you think could be made. Thanks!

**Pull request closed without merging**